kythe/cxx/indexer/proto/file_descriptor_walker.cc

```diff
@@ -358,6 +371,17 @@ void FileDescriptorWalker::VisitField(const std::string* parent_name,
     }
     const std::vector<int>& type_span = location_map_[lookup_path];
     InitializeLocation(type_span, &type_location);
+
+    // If we're in a message or enum type, decorate only the span
+    // covering the type name itself, not the full package name.
```

Comment:
Can you add that we are doing this so the type name doesn't span more than 1 line so it is easier for UIs to display?